pages/role/apply/views/index.jsx

```diff
@@ -27,11 +28,24 @@ const formatters = {
   }
 };
 
-const Page = () => {
+const Page = ({ addRoleTasks, schema }) => {
+  if (schema.type.options < 1) {
```

Comment:
This feels like it's missing a `.length`

Reply:
Also, if we mean `=== 0` can we say `=== 0`?

Reply:
sorted.